docs/backplane/requirements/acs/10-acs-managed-service.md

```diff
@@ -136,17 +140,13 @@ Permissions at cluster scope.
 * create pods/portforward
 * create pods/exec
 * delete pods
-* view clusterpackages
-* view clusterobjectdeployments
-* view clusterobjectsets
-* view clusterobjecttemplates
+* view addons
+* view addoninstance
+* view addonoperators
 * view packages
 * view objectdeployments
 * view objectsets
 * view objecttemplates
-* view addons
-* view addoninstance
-* view addonoperators
```

Comment:
Why is this removed from docs?  Premissions still exist at project scope.

Reply:
It's only moved. There are two lists: namespace and cluster scopes